Packs/Reco/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,16 @@
+
+#### Integrations
+
+##### Reco
+
+- Get sensitive assets publicly exposed 
+- Get files shared with 3rd parties 
+- reco-get-sensitive-assets-with-public-link
+- reco-get-3rd-parties-accessible-to-data-list
+- reco-get-files-shared-with-3rd-parties
+
+#### Playbooks
+
+##### New: Reco-Reduce-Risk-Google-Publicly-Exposed-Files
```

Comment:
The playbook name is "Reco - Reduce Risk - Google Publicly Exposed Files"
please update